src/Workspaces/SharedUtilitiesAndExtensions/Compiler/VisualBasic/Extensions/SyntaxTriviaExtensions.vb

```diff
@@ -40,7 +40,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions
             Return trivia.Kind = SyntaxKind.CommentTrivia OrElse trivia.Kind = SyntaxKind.DocumentationCommentTrivia
         End Function
 
-        <Extension()>
```

Comment:
pls consider undo the change unrelated to the fix, or at least have them in a separate commit to keep history clean (although I have to admit we have been doing a terrible job on this ourselves :P)